App.js

```diff
@@ -11,23 +15,56 @@ import rootReducer from './reducers'
 
 const store = createStore(rootReducer, composeWithDevTools(applyMiddleware(thunk)))
 
+const AppNavBar = createBottomTabNavigator(
+  {
+    Home: HomeScreen,
+    Profile: ProfilePage,
+    Messages: MessagesScreen,
+    Schedule: ScheduleScreen
+  },
+  {
+    defaultNavigationOptions: ({ navigation }) => ({
+      tabBarIcon: ({ focused, horizontal, tintColor }) => {
+        const { routeName } = navigation.state;
+        let iconName;
+        if (routeName === 'Home') {
+          iconName = 'home'
+        } else if (routeName === 'Profile') {
+          iconName = 'user'
+        } else if (routeName === 'Messages') {
+          iconName = 'message-circle'
+        } else if (routeName === 'Schedule') {
+          iconName = 'calendar'
+        }
+
```

Comment:
This looks good, I'm wondering if there is a way to refactor into a switch statement later to dry it up a bit.